lib/CapabilityDelegation.js

```diff
@@ -145,3 +158,12 @@ module.exports = class CapabilityDelegation extends ControllerProofPurpose {
     return proof;
   }
 };
+
+// the capability must be fetched in the security context
```

Comment:
@dlongley aside from adding a few more comments, I added this little helper since we were working on this together.